apps/system/js/remote/remote_app_window_manager.js

```diff
@@ -59,6 +64,20 @@
       });
     },
 
+    launchPresentationApp: function(config, immediate) {
+      this.debug('launch presentation app...');
+
+      this.requestId = config.requestId;
+      if (!config.manifest.permissions ||
+          !config.manifest.permissions.presentation) {
```

Comment:
Because I think `_sendPresentationResult()` should be just in one file. If we put permission checking in `multi_screen_controller.js`, we also need to put `_sendPresentationResult()` in that.